tuplex/core/include/ExceptionInfo.h

```diff
@@ -0,0 +1,43 @@
+//--------------------------------------------------------------------------------------------------------------------//
+//                                                                                                                    //
+//                                      Tuplex: Blazing Fast Python Data Science                                      //
+//                                                                                                                    //
+//                                                                                                                    //
+//  (c) 2017 - 2021, Tuplex team                                                                                      //
+//  Created by Benjamin Givertz first on 1/1/2022                                                                     //
+//  License: Apache 2.0                                                                                               //
+//--------------------------------------------------------------------------------------------------------------------//
+
+#ifndef TUPLEX_EXCEPTIONINFO_H
+#define TUPLEX_EXCEPTIONINFO_H
+
+namespace tuplex {
+    /*!
+     * Struct to hold information that maps input partitions to input exceptions that occur within them.
+     *
+     * Explanation:
+     * Each input partition is passed the same vector of all input exceptions that occured during data parallelization
+     * or caching. Thus, each input partition is must know how many input exceptions occur in its partition, the index
+     * of the input exception partition where its first exception occurs, and the offset into that partition where the
+     * first exception occurs. These values are held in this struct and each input partition is mapped to an ExceptionInfo.
+     */
+    struct ExceptionInfo {
+        size_t numExceptions; //! number of exceptions that occur within a single input partition
```

Comment:
so this is the count of exception rows? or the count of exception partitions?

Reply:
Its rows, clarified this